x-pack/plugins/cases/public/components/actions/tags/edit_tags_flyout.test.tsx

```diff
@@ -33,63 +34,56 @@ describe.skip('EditTagsFlyout', () => {
   };
 
   beforeEach(() => {
-    appMock = createAppMockRenderer();
     jest.clearAllMocks();
+    appMock = createAppMockRenderer();
+
+    useGetTagsMock.mockReturnValue({ isLoading: false, data: tags });
```

Comment:
Just thinking out loud but if the `beforeEach` is taken into account for the timeout of a test maybe this doesn't need to be done every time?

Reply:
Good point! It is worth a try. I think the `appMock = createAppMockRenderer();`, right?

Reply:
Dunno, some tests have it in the main class iirc 🤷